controller/templates/controller/index.html

```diff
@@ -25,6 +25,12 @@
     }
   </style>
 {% endblock extra_css %}
+{% block extra_js %}
+  <link rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/@shoelace-style/shoelace@2.18.0/cdn/themes/light.css" />
+  <script type="module"
+          src="https://cdn.jsdelivr.net/npm/@shoelace-style/shoelace@2.18.0/cdn/shoelace-autoloader.js"></script>
+{% endblock extra_js %}
```

Comment:
We will need local copies of the js and css files for deployment.

Reply:
I've mentioned it before and I say it here again - manually coping JS files to the repo I think is a bad idea. It's a recipe for getting those components out of sync and outdated. We should use a proper dependency management for JS packages, like `npm`, now that it seems we have a few.

Reply:
Besides, I don't know how to download this locally and ship it with the software. Shoelace seems to be a complex tool with multiple components and style files, not a simple, self-contained JS file.

I'd really suggest to leave this as it is an open an issue to move to using `npm` (or other tool) for JS dependency management, now that it seems we are going to have a gap until there're more things to do in the project.

Reply:
Sure, no problem. Just making aware that they will not be able to be remotely sourced in deployment. Let's leave it then.

Reply:
Add #275 for this